linux-support.md

```diff
@@ -0,0 +1,55 @@
+# .NET Support and Compatibility for Linux Distributions
+
+.NET is officially supported on multiple [Linux distributions](os-support-lifecycle.md) and [supported by the community on several others](linux.md).
```

Comment:
> officially supported

I would avoid "officially" in these docs. It feels superfluous and/or ambiguous. 

- Officially supported -> Supported by Microsoft
- Official packaging guidelines -> packaging guidelines. (We do not have non-official packaging guidelines.)